docs/language.rst

```diff
@@ -189,11 +190,34 @@ in Kestrel are mutable. They can be partially updated, e.g., new attributes
 added through an analytics, and they can be overwritten by a variable
 assignment to an existing variable.
 
+
+Variable Transforms
+===================
+
+TIMESTAMPED
+-----------
+
+STIX data from STIX-Shifter_ comes in the form of "observations," or the
+``observed-data`` STIX Domain Object (SDO).  This object contains (or
+references) SCOs, along with a time range for when those observables were
+actually seen.
+
+SCOs themselves do not contain any information for when they were observed,
+but you can retrieve the timestamps from those observations by using the
```

Comment:
maybe: but you can retrieve SCOs with timestamps from those observations by using the